bindata/network/additional-networks/crd/001-crd.yaml

```diff
@@ -48,6 +48,9 @@ spec:
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+  creationTimestamp: null
```

Comment:
Surprised to see `creationTimestamp`

Reply:
That I got by running the script in whereabouts to generate the files
https://github.com/openshift/whereabouts-cni/blob/master/hack/generate-code.sh